src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -127,10 +120,15 @@ class RA_GUIBASE_API KeyMappingManager
                                            const std::string& wheelString,
                                            const std::string& actionString );
 
-    // Maybe there's a better way to get enum value from string, even without Q_ENUM
-    // defined for Qt::KeyboardModifier and Qt::MouseButton ?
+    /// Return KeyboardModifiers described in modifierString, multiple modifiers
+    /// are comma separated in the modifiers string, as in
+    /// "ShiftModifier,AltModifier". This function do note trim any white space.
```

Comment:
Typo here `note` -> `not`
Why giving an example here but not in the doc of `enumNamesFromMouseButtons` and `enumNamesFromKeyboardModifiers`?

Reply:
well, do you want to give an example of Qt::MouseButtons ? like "Qt::MiddleButton" ? 
For me, the example for a string parameter seems useful while for an enum it's useless.

Reply:
Nevermind, I didn't see that the string is an input